src/home/components/SignupContents/EmailForm/EmailForm.tsx

```diff
@@ -33,7 +33,7 @@ export const EmailForm = ({ onConfirm }: EmailFormProps) => {
             value={email}
             onChange={onChange}
             isNegative={!!emailError}
-            helperLabel={emailError}
+            helperLabel={emailError ? emailError : undefined}
```

Comment:
undefined 말고 빈값으로 둔다면 다음처럼 수정해도 될 것 같아요~
```suggestion
            helperLabel={emailError ?? ''}
```

Reply:
빈 값을 그대로 보내도 해결되는 거 같습니다!